app/views/passengers/show.html.erb

```diff
@@ -0,0 +1,45 @@
+<h2>Passenger Info:</h2>
+
+<section>
+  Passenger ID:
+  <%= @passenger[:id] %><br>
```

Comment:
You should not be using the hash-like syntax for retrieving attribute values, you should be using the built-in "dot methods". So this should be `@passenger.id` instead.